source/integration-guide/order-confirmation.rst

```diff
@@ -206,7 +206,7 @@ sizeAlias
         // When the size is not understandable by customers, like:
         size: "2148"
         // Then the sizeAlias can be use to specify what the customer will see for
-        // this product in his purchase history the next time they open the widget:
+        // this product in their purchase history the next time they open the widget:
```

Comment:
+1 for using they and their. http://www.wsj.com/articles/can-they-be-accepted-as-a-singular-pronoun-1428686651